packages/jest-jasmine2/src/jasmine-check-install.js

```diff
@@ -0,0 +1,29 @@
+// Initialize jasmine-check within the vm context:
+jest.unmock('jasmine-check');
+require('jasmine-check').install();
+
+// Replace the 'check' functions with ones that default to options from
+// the jest configuration:
+(function() {
+  const configOptions = jasmine.testcheckOptions;
+  function makeMergeOptions(object, methodName) {
+    const original = object[methodName];
+    object[methodName] = function(specName, options, argGens, propertyFn) {
+      if (!propertyFn) {
+        propertyFn = argGens;
+        argGens = options;
+        options = {};
+      }
+      const mergedOptions = Object.assign({}, configOptions, options);
+      return original(specName, mergedOptions, argGens, propertyFn);
+    };
+  }
+
+  makeMergeOptions(check, 'it');
+  makeMergeOptions(check, 'iit');
+  makeMergeOptions(check.it, 'only');
+  makeMergeOptions(check, 'fit');
+  makeMergeOptions(check, 'xit');
+})();
+
+jasmine.testcheckOptions = undefined;
```

Comment:
I think we should use `delete` here and maybe do this in the jasmine2 file right after the requireModule call? That way the assignment and deletion happens in the same file.